recipes/server.rb

```diff
@@ -23,8 +23,8 @@
 include_recipe 'rsyslog::default'
 
 directory node['rsyslog']['log_dir'] do
-  owner    'root'
-  group    'root'
+  owner    node['rsyslog']['user'] ? node['rsyslog']['user']  : 'root'
```

Comment:
Probably better to just make this `node['rsyslog']['user']` rather than using if-then logic because that attribute well always be set (since it is defined in `attributes/default.rb`)